build-tools/automation/azure-pipelines.yaml

```diff
@@ -453,80 +453,6 @@ stages:
 
     - template: yaml-templates/fail-on-issue.yaml
 
-  # MSBuild Smoke Tests - Mac
-  - template: yaml-templates/run-msbuild-tests.yaml
-    parameters:
-      testOS: macOS
-      jobName: mac_smoke_msbuild_tests
-      jobDisplayName: macOS > Tests > MSBuild
-      agentCount: 2
-      testFilter: cat = SmokeTests
-      installApkDiff: true
-
-  # MSBuild Smoke Tests - Win
-  - template: yaml-templates/run-msbuild-tests.yaml
-    parameters:
-      testOS: Windows
-      jobName: win_smoke_msbuild_tests
-      jobDisplayName: Windows > Tests > MSBuild
-      agentCount: 2
-      testFilter: cat = SmokeTests
-      installApkDiff: true
```

Comment:
I believe you are referring to the Windows build + run some in-tree tests on it, which is this stage:

https://github.com/xamarin/xamarin-android/blob/d26558092b0fe5edb6974bb8e59e317f147edfa6/build-tools/automation/azure-pipelines.yaml#L90

It will still run here.  The tests that were previously built by the job(s) you referenced are simply moved to the `MSBuild` stage `Windows` test.

Reply:
The windows build stage (and testing of in-tree build artifacts rather than nugets) is part of https://github.com/xamarin/xamarin-android/blob/452e742e214cad42f2f30ea74097e422e49c04e3/build-tools/automation/yaml-templates/build-windows.yaml and will still be covered. This job was only running SmokeTests from Xamarin.Android.Build.Tests